src/Calypso-Browser/ClyBrowserSearchDialogWindow.class.st

```diff
@@ -102,7 +102,7 @@ ClyBrowserSearchDialogWindow >> openOn: aQuery withTitle: dialogTitle thenDo: re
 	self showItemsOf: aQuery.
 	self title: dialogTitle.
 
-	browser openModal: self.
+	self openModal.
```

Comment:
I can't load full PR right now. How it changes the behavior? 
Previously simple #openModal behaves in the way that you can lost it by clicking on the initial browser.

Reply:
Hi @dionisiydk.
> How it changes the behavior?
> Previously simple #openModal behaves in the way that you can lost it by clicking on the initial browser.

Right now what will happen is that the window will open.
And the caller method will wait until it is closed.
Clicking outside the window does not close the window, and does not make the modality return.

At least that happens for most windows. A window could closed itself when it loses focus. We don't have power there :).

The main impact here is that the browser will continue answering to events.
I can fix that in this PR, no problem, but I'd need you to tell me how to get the browser window => because modality is a window concern.

Reply:
```Smalltalk
browser window
```

Reply:
Fixed :) Thanks for the review.